### PR TITLE
Upgrade to polkadot-v1.5.0

### PR DESCRIPTION
**Pull request by jasl**
Breaking change:
- https://github.com/paritytech/polkadot-sdk/pull/1256
  - @nanometerzhu Please help to migrate to new way, I keep the deprecation this time

**IMPORTANT**:
- https://github.com/paritytech/polkadot-sdk/pull/1661
  - @nanometerzhu This requires a migration, we can skip for PoC6, but must do on Khala & Phala

### CODE REVIEW

**Comment by jasl** [comment]
```
error[E0432]: unresolved import `sp_consensus_grandpa::VersionedAuthorityList`
  --> standalone/pherry/src/lib.rs:23:50
   |
23 | use sp_consensus_grandpa::{AuthorityList, SetId, VersionedAuthorityList};
   |                                                  ^^^^^^^^^^^^^^^^^^^^^^ no `VersionedAuthorityList` in the root

For more information about this error, try `rustc --explain E0432`.
error: could not compile `pherry` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
```

@kvinwang

File: standalone/node/src/command.rs [hunk 1]
```diff
@@ -255,34 +254,6 @@ pub fn run() -> sc_cli::Result<()> {
                 Ok((cmd.run(client, backend, Some(aux_revert)), task_manager))
             })
         }
-        #[cfg(feature = "try-runtime")]
```
Comment (h4x3rotab) [inline standalone/node/src/command.rs:258]: Is `try-runtime` completely removed?

**Review by h4x3rotab — APPROVED** [review]

File: standalone/node/src/command.rs (hunk 1 above)
Comment (jasl) [reply standalone/node/src/command.rs:258]: Moved to https://github.com/paritytech/try-runtime-cli

**Review by kvinwang — APPROVED** [review]
LGTM

**Pull request merged**